website/content/api-docs/system/secrets-sync.mdx

```diff
@@ -237,9 +250,25 @@ fallbacks on the AWS credentials provider chain and tries to infer authenticatio
 - `secret_access_key` `(string: "")` - Secret access key to authenticate against the AWS secrets manager. If omitted,
 authentication fallbacks on the AWS credentials provider chain and tries to infer authentication from the environment.
 
+- `role_arn` `(string: "")` - Specifies a role to assume when connecting to AWS. When assuming a role, Vault uses temporary
+STS credentials to authenticate. An initial session with the proper trust relationship must exist for Vault to be able to
+assume this role. The role can be in a different account.
+
+- `external_id` `(string: "")` - Optional extra protection that must match the trust policy granting access to the AWS IAM
+role ARN. We recommend using a different random UUID per destination.
+
 - `region` `(string: "")` - Region where to manage the secrets manager entries. If omitted, configuration fallbacks on
 the AWS credentials provider chain and tries to infer region from the environment.
 
+- `secret_name_template` `(string: "")` - Template to use when generating the secret names on the external system.
+The default template yields names like `vault/kv_1234/my-secret`. See [this documentation](/vault/docs/sync#name-template) for more details.
+
+- `custom_tags` `(map<string|string>: nil)` - List of custom tags or labels to set on the synced secrets at the destination.
+Custom tags are merged with internal built-in tags.
+
+- `granularity` `(string: "secret-path")` - Determines what level of information is synced as a distinct resource at the
+destination. See [this documentation](/vault/docs/sync#granularity) for more details.
+
```

Comment:
I copied the description from the BE @raymonstah used to document the request payload. "Level of information" sounded right to me but it could be good to get external input since our perspective is tainted by knowing how it works under the hood.